sale_advance_payment/tests/test_sale_advance_payment.py

```diff
@@ -21,13 +22,13 @@ def setUpClass(cls):
 
         # Products
         cls.product_1 = cls.env["product.product"].create(
-            {"name": "Desk Combination", "invoice_policy": "order"}
+            {"name": "Desk Combination", "type": "product", "invoice_policy": "order"}
```

Comment:
I would remove the "type": "product", from the products. This crashes the tests if you have a clean database with only the sale_advance_payment (as the only dependency is sale).
We did this modification in another PR recently. Perhaps the code is not updated with 14.0. Maybe there are some other conflicts. https://github.com/OCA/sale-workflow/pull/1847

Reply:
Hi @MateuGForgeFlow,

Update change as you suggest, Please check,

Thanks

Reply:
Hi @JasminSForgeFlow! There's still the "type": "service" for product 3. Thank you!

Reply:
Hi @MateuGForgeFlow ,
Update changes for 3rd product.
Please check.
Thanks

Reply:
Thank you! @JasminSForgeFlow